pkg/apimodel/grant.go

```diff
@@ -2,16 +2,16 @@ package apimodel
 
 // CreateGrant represents the request object of creating a grant
 type CreateGrant struct {
-	Domain         string   `json:"domain"`
-	AllowedActions []string `json:"allowed_actions"`
+	Domain             string   `json:"domain" validate:"fqdn"`
```

Comment:
To triple check, will this allow *?

Reply:
I was thinking of making it the same as cookie sharing cross-domain rule.

E.g.

```
example.com
```

gives you the access to 
```
foo.example.com
bar.example.com
```

And still makes it compatible as https://en.wikipedia.org/wiki/Fully_qualified_domain_name

Reply:
`*` is no not valid, the regex is defined here https://github.com/go-playground/validator/blob/d6b17fd90bd4dd9d16a594c3035ceadc3de0193a/regexes.go#L41